make/modules/java.base/Copy.gmk

```diff
@@ -277,3 +277,18 @@ $(eval $(call SetupTextFileProcessing, CREATE_CLASSFILE_CONSTANTS_H, \
 TARGETS += $(CREATE_CLASSFILE_CONSTANTS_H)
 
 ################################################################################
+
+################################################################################
+# Copy mallochook.h
+
+ifeq ($(call isTargetOs, windows), false)
```

Comment:
This tests for 'not windows', but in the next file you test for 'is Linux or macOS'.  What about AIX we have this now too in Sapmachine ?  I think the makefile checks should be consistent (I guess test for 'is Linux or macOS') .

Reply:
Initially I included AIX as well, but removed it since. I've modified the test to Linux and MacOSX